config/spinnaker.yml

```diff
@@ -294,3 +294,18 @@ providers:
       name: my-docker-registry-account
       address: https://index.docker.io/
       repository: library/nginx
+
+  openstack:
+    # For more information on configuring Openstack, see
+    # TODO Added doc link when available
+
+    enabled: false
+    primaryCredentials:
+      name: my-openstack-account
+      authUrl:
+      username:
+      password:
+      projectName:
+      domainName:
```

Comment:
In default-spinnaker-local you gave "Domain" as a default value. Did you mean to have that here?